.buildkite/scripts/snyk/report.sh

```diff
@@ -139,5 +179,5 @@ REPOSITORY_BASE_URL="docker.elastic.co/logstash/"
 for TARGET_BRANCH in "${TARGET_BRANCHES[@]}"
 do
   echo "Using $TARGET_BRANCH branch for docker images."
-  resolve_version_and_report_docker_images "$TARGET_BRANCH"
+  #resolve_version_and_report_docker_images "$TARGET_BRANCH"
```

Comment:
review note: intentionally disabling since we are at early stage and dashboard is frozen due to this.